### PR TITLE
Create DataUsageAgreement

### PR DESCRIPTION
**Pull request by EvaRuthM**
I tried to add our data usage agreement to the CMS but wasn't able to style it, even when typing in markdown commands. Any help on what I was doing wrong? 

another option for this – I can upload the PDF of the doc

### CODE REVIEW

**Comment by nholden** [comment]
> I tried to add our data usage agreement to the CMS but wasn't able to style it, even when typing in markdown commands. Any help on what I was doing wrong?

Adding the data usage agreement to the CMS would add it to our website, not to this repo. Maybe we can chat about where the data usage agreement should live in another issue?

What do you think about saving this as a markdown file (`.md`)? That would make it a little easier to read in GitHub.

Is the block with our name, address, and URL at the top a legal requirement? Wondering how people would use this.

File: DataUsageAgreement.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Texas Justice Initiative     
+P.O. Box 164286  
+Austin, TX 78716   
+https://www.texasjusticeinitiative.org  
+
+![](https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg)
```
Comment (nholden) [inline DataUsageAgreement.md:6]:
```suggestion
<img src="https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg" alt="TJI logo" style="max-width:200px;">
```

We could do something like this if we wanted to make the logo more reasonably sized in the rendered version.

File: DataUsageAgreement.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## 1. Limited License
+By downloading or accessing any Datasets, you hereby agree to all of the terms specified in this license. TJI grants  Recipient the limited right to use the Datasets only for the Recipient’s own academic, nonprofit, and research purposes. The Recipient may not use TJI’s Datasets for any commercial or profit-making purpose; and may only use the Dataset in accordance with this Agreement. TJI retains all rights, title, and interest in and to the Dataset, and does not grant any rights in the Dataset other than as specified herein.
+
+## 2. Attribution 
+When using TJI Datasets in any reports, articles, papers, journals or other mediums you must document and give TJI (appropriate credit)[https://creativecommons.org/licenses/by/3.0/us/#], provide a link to TJI’s website, and specifically indicate what, if any, changes were made to the Datasets. You may do so in any reasonable manner, but you may not in any way  suggest that TJI endorses you or your use of its Datasets.
```
Comment (nholden) [inline DataUsageAgreement.md:18]:
```suggestion
When using TJI Datasets in any reports, articles, papers, journals or other mediums you must document and give TJI [appropriate credit](https://creativecommons.org/licenses/by/3.0/us/#), provide a link to TJI’s website, and specifically indicate what, if any, changes were made to the Datasets. You may do so in any reasonable manner, but you may not in any way  suggest that TJI endorses you or your use of its Datasets.
```

I swap the parentheses and square brackets in Markdown links by accident every day 😆

**Review by nholden — APPROVED** [review]
Looks great! Added a couple suggestions.

File: DataUsageAgreement.md (hunk 2 above)
Comment (EvaRuthM) [reply DataUsageAgreement.md:18]: thanks for catching this!!!

File: DataUsageAgreement.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Texas Justice Initiative     
+P.O. Box 164286  
+Austin, TX 78716   
+https://www.texasjusticeinitiative.org  
+
+<img src="https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg" alt="TJI logo" style="max-width:200px;">
```
Comment (nholden) [inline DataUsageAgreement.md:6]:
```suggestion
<img src="https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg" alt="TJI logo" width="200px">
```

My last suggestion didn't work as expected. Try this? 😅

Comment (nholden) [reply DataUsageAgreement.md]: Much better 🙌

**Pull request merged**